From 48f637b51c39efa5c43bb28aa02e7ea093287d72 Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Fri, 23 Aug 2024 15:38:30 +0800
Subject: [PATCH 2/2] FROM AOSC: fix format error
X-Developer-Signature: v=1; a=openpgp-sha256; l=1748; i=xtexchooser@duck.com;
 h=from:subject; bh=/vxjaLchV3cYmX9iOtHfL3/8UTCgQDuRFOp4gLIfic8=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDGkn7DLven1xeKTF+jfn0qYjromyG/Wqzha4q7ccFZ7tt
 XJT6smEjlIWBjEuBlkxRZYiwwZvVp10ftFl5bIwc1iZQIYwcHEKwETq1Rn+F2w2/ma2UurClkdV
 y8XeC7v7XDD61a/02EnguwZ/aHhoCSPDjEPMih5fz6v8dGNMFK/nyfg2+aW0LacjT+/CNaI81/v
 ZAA==
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

I am not familiar with Vala at all.
I don't know how to fix the "format not a string literal, argument types not checked" compilation error,
I even can't find a exisiting patch for this.
So I have to remove these code. I made the choice for you guys.
---
 libgitg/gitg-repository-list-box.vala | 17 +----------------
 1 file changed, 1 insertion(+), 16 deletions(-)

diff --git a/libgitg/gitg-repository-list-box.vala b/libgitg/gitg-repository-list-box.vala
index df545035f198..854c48f5c576 100644
--- a/libgitg/gitg-repository-list-box.vala
+++ b/libgitg/gitg-repository-list-box.vala
@@ -561,22 +561,7 @@ namespace Gitg
 
 		private void delete_source_clicked(File workdir, string uri, Row row)
 		{
-			var alert_dialog = new Gtk.MessageDialog((Gtk.Window) row.get_toplevel(),
-					Gtk.DialogFlags.MODAL, Gtk.MessageType.ERROR, Gtk.ButtonsType.NONE,
-					_("Deleting repository source files “%s” will delete them from disk and cannot be undone. Are you sure?"), uri);
-			alert_dialog.add_button(_("Cancel"), DeleteSources.CANCEL);
-			alert_dialog.add_button(_("Move to trash"), DeleteSources.TRASH);
-			alert_dialog.add_button(_("Delete permanently"), DeleteSources.DELETE);
-
-			var delete_button = alert_dialog.get_widget_for_response(DeleteSources.DELETE);
-			delete_button.get_style_context().add_class(Gtk.STYLE_CLASS_DESTRUCTIVE_ACTION);
-
-			alert_dialog.response.connect ((id) => {
-				handle_delete_source_response(id, workdir, uri, row);
-				alert_dialog.destroy();
-			});
-
-			alert_dialog.run();
+			handle_delete_source_response(DeleteSources.DELETE, workdir, uri, row);
 		}
 
 		private void handle_delete_source_response(int id, File workdir, string uri, Row row)
-- 
2.46.0

